src/views/new-version-overlay.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const React = require('react');
+const { createContainer } = require('sovereign');
+const Button = require('react-material/components/Button');
+
+const Overlay = require('../components/overlay');
+const OverlayTitle = require('../components/overlay-title');
+const OverlayFooter = require('../components/overlay-footer');
+
+const contentStyle = {
+  position: 'relative'
+};
+
+const releaseNotesLink = 'https://github.com/parallaxinc/Parallax-IDE/releases';
+const installationNotesLink = 'https://www.parallax.com/downloads/parallax-ide-chrome';
+
+class NewVersionOverlay extends React.Component {
+
+  constructor(...args){
+    super(...args);
+  }
+
+  render(){
+
+    const {
+      handlers
+    } = this.props;
+
+    const {
+      hideOverlay
+    } = handlers;
+
+    const {
+      name,
+      version
+    } = chrome.runtime.getManifest();
+
+    return (
+      <Overlay>
+        <OverlayTitle>{name} Automatically Updated to v{version}</OverlayTitle>
+        <div style={contentStyle}>
+          You are now running a new release of {name}!
+        </div>
+        <div style={contentStyle}>
+          See <a href={releaseNotesLink} target="_blank">Release Notes</a> and <a href={installationNotesLink} target="_blank">Installation Notes</a> for more details.
+        </div>
+        <OverlayFooter>
+          <Button onClick={hideOverlay}>OK</Button>
+        </OverlayFooter>
+      </Overlay>
+    );
+  }
+}
+
+module.exports = createContainer(NewVersionOverlay, {
```

Comment:
`createContainer` isn't being used here, remove it and just export the react component